src/components/Links/Links.test.jsx

```diff
@@ -1,33 +1,35 @@
 import React from 'react';
-import renderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
 
 import Links from './Links';
 
 const linksData = {
-  title: 'examples',
-  links: [
-    {
-      title: 'I am first',
-      href: 'https://example1.com',
-    },
-    {
-      title: 'I am second',
-      href: 'https://example2.com',
-    },
-    {
-      title: 'I am third',
-      href: 'https://example3.com',
-    },
-  ],
+  id: 'examples',
+  title: 'Examples',
+  links: [...new Array(8)].map((_, index) => ({
+    title: `Test ${index + 1}`,
+    href: `http://example.com/${index + 1}`,
+  })),
 };
 
 describe('<Links />', () => {
-  it('renders correctly', () => {
-    const tree = renderer
-      .create((
-        <Links title={linksData.title} links={linksData.links} />
-      ))
-      .toJSON();
-    expect(tree).toMatchSnapshot();
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = shallow((
+      <Links id={linksData.id} title={linksData.title} links={linksData.links} />
+    ));
+  });
+
```

Comment:
Do we need a test where we have less than 5 links?

Also in the below tests, do we need to compare `show less`/`show all ...` button texts?

Reply:
I can add assertions to verify the button text for the "show less / show all" button as well as a test for less than 5 links.